### PR TITLE
[Docs] Change the process in all files to workflow

### PR DESCRIPTION
**Pull request by wushanru**
Change the process in all files to workflow 

17c823f98b8bd6ba358adf113db86a3

### CODE REVIEW

File: docs/docs/en/guide/project/project-list.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 | Workflow Relation   | Shows dynamic status of all the workflows in a project.                                                                                                                                                                                                                                                                   |
 | Task                | Task is a discrete action in a Workflow. Apache DolphinScheduler supports SHELL, SQL, SUB_PROCESS (sub-process), PROCEDURE, MR, SPARK, PYTHON, DEPENDENT ( depends), and plans to support dynamic plug-in expansion, (SUB_PROCESS). It is also a separate process definition that can be started and executed separately. |
 | Task Instance       | Instantiation of the task node in the process definition, which identifies the specific task execution status.                                                                                                                                                                                                            |
```
Comment (SbloodyS) [inline docs/docs/en/guide/project/project-list.md]: Can you modify the process definition of the whole file?

**Review by SbloodyS — COMMENTED** [review]
BTW, it's better to use english in the commit message.

**Comment by EricGao888** [comment]
@wushanru Thanks for submitting this PR. There are multiple places where we need to replace process with definition, could u plz also fix them in this PR? Just simply ctr/cmd + shift + r in your IDE. BTW, instead of giving a WeChat screenshot, could u plz write some text description when submitting PR? Thanks again : )

**Comment by fuchanghai** [comment]
image
@wushanru There are still some changes that need to be made。plz global search

**Comment by EricGao888** [comment]
> image @wushanru There are still some changes that need to be made。plz global search

+1, we'd better keep the docs consistent.

File: docs/docs/en/contribute/join/become-a-committer.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ rather than unnecessary constraints.
 
 Being a committer enables you to more easily make 
 changes without needing to go through the patch 
-submission process. 
+submission workflow.
```
Comment (SbloodyS) [inline docs/docs/en/contribute/join/become-a-committer.md]: This should not be modified.

File: docs/docs/en/contribute/join/become-a-committer.md [hunk 3]
```diff
@@ -82,7 +82,7 @@ receive a follow-up message with the next steps for
 establishing you as a committer.
 ```
 
-## Invitation acceptance process
+## Invitation acceptance workflow
```
Comment (SbloodyS) [inline docs/docs/en/contribute/join/become-a-committer.md]: Same here.

File: docs/docs/en/contribute/join/become-a-committer.md [hunk 4]
```diff
@@ -133,8 +133,8 @@ As a committer, you could
 2. Create and push codes to the new branch in the Apache repo.
 3. Prepare a new release. Remember to confirm with the committer team before you prepare, because that it is the right time to create the release.
 
-The PMC hopes that the new committer will take part in the release process as well as release voting, even though their vote will be regarded as `+1 no binding`.
-Being familiar with the release process is key to being promoted to the role of PMC member.
+The PMC hopes that the new committer will take part in the release workflow as well as release voting, even though their vote will be regarded as `+1 no binding`.
+Being familiar with the release workflow is key to being promoted to the role of PMC member.
```
Comment (SbloodyS) [inline docs/docs/en/contribute/join/become-a-committer.md]: Same here.

File: docs/docs/en/contribute/join/become-a-committer.md [hunk 5]
```diff
@@ -150,7 +150,7 @@ The V.P. and chair of the PMC is the secretary, who is responsible for initializ
 
 In most cases, a new PMC member is nominated from the committer team. But it is also possible to become a PMC member directly, so long as the PMC agrees to the nomination and is confident that the candidate is ready. For instance, this can be demonstrated by the fact that he/she has been an Apache member, an Apache officer, or a PMC member of another project.
 
-The new PMC voting process should also follow the `[DISCUSS]`, `[VOTE]` and `[RESULT][VOTE]` procedures using a private mail list, just like the voting process for new committers.
+The new PMC voting workflow should also follow the `[DISCUSS]`, `[VOTE]` and `[RESULT][VOTE]` procedures using a private mail list, just like the voting workflow for new committers.
```
Comment (SbloodyS) [inline docs/docs/en/contribute/join/become-a-committer.md]: Same here.

File: docs/docs/en/contribute/join/become-a-committer.md [hunk 6]
```diff
@@ -16,14 +16,14 @@ Community involvement: Committers should have a constructive and friendly attitu
 
 ## Nominate new committer
 
-In Dolphinscheduler, **new committer nomination** could only be officially started by existing PMC members. If a new committer feels that he/she is qualified, he/she should contact any existing PMC member and discuss. If this is agreed among some members of the PMC, the process will kick off.
+In Dolphinscheduler, **new committer nomination** could only be officially started by existing PMC members. If a new committer feels that he/she is qualified, he/she should contact any existing PMC member and discuss. If this is agreed among some members of the PMC, the workflow will kick off.
```
Comment (SbloodyS) [inline docs/docs/en/contribute/join/become-a-committer.md]: Same here.

**Review by fuchanghai — APPROVED** [review]

**Comment by fuchanghai** [comment]
@SbloodyS  i'm not found dead link . can you re-run?

**Comment by SbloodyS** [comment]
> @SbloodyS i'm not found dead link . can you re-run?

There are some conflicts needs to be resolved. @wushanru

**Comment by wushanru** [comment]
> > @SbloodyS i'm not found dead link . can you re-run?
> 
> There are some conflicts needs to be resolved. @wushanru

I don't quite understand what you mean. What should I do

**Comment by wushanru** [comment]
> > @SbloodyS i'm not found dead link . can you re-run?
> 
> There are some conflicts needs to be resolved. @wushanru

I don't quite understand what you mean. What should I do。@SbloodyS

**Comment by wushanru** [comment]
> > @SbloodyS i'm not found dead link . can you re-run?
> 
> There are some conflicts needs to be resolved. @wushanru

I don't quite understand what you mean. What should I do @SbloodyS

**Comment by EricGao888** [comment]
> > > @SbloodyS i'm not found dead link . can you re-run?
> > 
> > 
> > There are some conflicts needs to be resolved. @wushanru
> 
> I don't quite understand what you mean. What should I do @SbloodyS



> > > @SbloodyS i'm not found dead link . can you re-run?
> > 
> > 
> > There are some conflicts needs to be resolved. @wushanru
> 
> I don't quite understand what you mean. What should I do @SbloodyS

@wushanru As you could see below, your branch has conflicts with the dev branch, therefore, you need to use `git rebase` command to resolve the conflicts. 
image

**Comment by wushanru** [comment]
> > > > @SbloodyS i'm not found dead link . can you re-run?
> > > 
> > > 
> > > There are some conflicts needs to be resolved. @wushanru
> > 
> > 
> > I don't quite understand what you mean. What should I do @SbloodyS
> 
> > > > @SbloodyS i'm not found dead link . can you re-run?
> > > 
> > > 
> > > There are some conflicts needs to be resolved. @wushanru
> > 
> > 
> > I don't quite understand what you mean. What should I do @SbloodyS
> 
> @wushanru As you could see below, your branch has conflicts with the dev branch, therefore, you need to use `git rebase` command to resolve the conflicts. image

I have resolved the conflict. Please take a look. Thank you @SbloodyS

**Comment by wushanru** [comment]
@

> > > > @SbloodyS i'm not found dead link . can you re-run?
> > > 
> > > 
> > > There are some conflicts needs to be resolved. @wushanru
> > 
> > 
> > I don't quite understand what you mean. What should I do @SbloodyS
> 
> > > > @SbloodyS i'm not found dead link . can you re-run?
> > > 
> > > 
> > > There are some conflicts needs to be resolved. @wushanru
> > 
> > 
> > I don't quite understand what you mean. What should I do @SbloodyS
> 
> @wushanru As you could see below, your branch has conflicts with the dev branch, therefore, you need to use `git rebase` command to resolve the conflicts. image

I haven't handled this open source code conflict. Can you tell me the detailed operation steps? thank! @SbloodyS

**Comment by fuchanghai** [comment]
@wushanru file named glossary had changed ,above conflict with you fix code

**Comment by wushanru** [comment]
> @wushanru file named glossary had changed ,above conflict with you fix code

@

> @wushanru file named glossary had changed ,above conflict with you fix code

@fuchanghai My first PR, this conflict is also the first time to resolve, I don't quite understand your meaning, please tell me the specific steps, thank you！

**Comment by wushanru** [comment]
> @wushanru file named glossary had changed ,above conflict with you fix code

@fuchanghai My first PR, this conflict is also the first time to resolve, I don't quite understand your meaning, please tell me the specific steps, thank you！

**Comment by EricGao888** [comment]
> > @wushanru file named glossary had changed ,above conflict with you fix code
> 
> @fuchanghai My first PR, this conflict is also the first time to resolve, I don't quite understand your meaning, please tell me the specific steps, thank you！

@wushanru Looks like you didn't correctly rebase your code and pushed the dev branch instead. Please refer to this link for instructions: https://github.com/sevntu-checkstyle/sevntu.checkstyle/wiki/Development-workflow-with-Git:-Fork,-Branching,-Commits,-and-Pull-Request

**Comment by EricGao888** [comment]
> > > @wushanru file named glossary had changed ,above conflict with you fix code
> > 
> > 
> > @fuchanghai My first PR, this conflict is also the first time to resolve, I don't quite understand your meaning, please tell me the specific steps, thank you！
> 
> @wushanru Looks like you didn't correctly rebase your code and pushed the dev branch instead. Please refer to this link for instructions: https://github.com/sevntu-checkstyle/sevntu.checkstyle/wiki/Development-workflow-with-Git:-Fork,-Branching,-Commits,-and-Pull-Request

BTW, the `master` branch in the above article is called `dev` branch in DolphinScheduler repo.

**Pull request closed without merging**